odt2txt/PKGBUILD

```diff
@@ -0,0 +1,44 @@
+# Maintainer: Alberto Fanjul <albertofanjul@gmail.com>
+
+source=(https://downloads.sourceforge.net/sourceforge/${pkgname}/${pkgname}-${pkgver}.tgz)
+sha256sums=('b297752910a404c1435e703d5aedb4571222bd759fa316c86ad8c8bbe58c6d1b')
+depends=('perl' 'unzip')
+
+package() {
+  cd ${srcdir}/${pkgname}-${pkgver}
+  make BINDIR=${pkgdir}/usr/bin installbin
+  make CONFIGDIR=${pkgdir}/etc installconfig
+  mv -f ${pkgdir}/usr/bin/docx2txt.sh ${pkgdir}/usr/bin/docx2txt
+}
+
+pkgname=odt2txt
+pkgver=0.5
+pkgrel=2
+pkgdesc="extracts the text out of OpenDocument Texts"
+url="https://github.com/dstosberg/odt2txt/"
+license=("GPL2")
+arch=('i686' 'x86_64')
+depends=('zlib-devel' 'libiconv-devel' 'mingw-w64-x86_64-libzip')
```

Comment:
You cannot mix and match MSYS2 packages (`zlib-devel`, `libiconv-devel`) and MINGW packages (`${MINGW_PACKAGE_PREFIX}-libzip`; you do not want to depend on 64-bit libzip when building 32-bit odt2txt).